content/cucumber/hooks.md

```diff
@@ -62,10 +83,61 @@ After do |s|
 end
 ```
 
+```java
+@After
+public void afterScenario(){
+    // Tell Cucumber to quit after this scenario is done - if it failed.
+    public void afterFailedScenario(Scenario scenario) {
+        if (scenario.isFailed()) {
+            // Do something when Scenario failed
+        }
+}
+```
+
+```javascript
+TODO: See Ruby or Java example
+```
+
+{{% text "javascript" %}}
+Hooks are used for setup and teardown the environment before and after each scenario.
```

Comment:
`for setup and teardown the environment` => `for setup and teardown of the environment`